pkg/dumper/query/dumper.go

```diff
@@ -47,16 +47,13 @@ func (d *textDumper) Dump(done chan<- struct{}, spec *config.Spec, concurrency i
 	for _, tbl := range tables {
 		var opts reader.ReadTableOpt
 
-		table := spec.Tables.FindByName(tbl)
-		if table == nil {
+		tableConfig := spec.Tables.FindByName(tbl)
+		if tableConfig == nil {
 			log.WithField("table", tbl).Debug("no configuration found for table")
 		}
 
-		if table != nil {
-			opts = reader.ReadTableOpt{
-				Limit:         table.Filter.Limit,
-				Relationships: d.relationshipConfigToOptions(table.Relationships),
-			}
+		if tableConfig != nil {
```

Comment:
since you are touching this part, maybe just add `else` here?

Reply:
done